builder/builder.go

```diff
@@ -1840,6 +1840,33 @@ func (b *Builder) SetURLRepo(name, url string) error {
 	return DNFConf.SaveTo(b.Config.Builder.DNFConf)
 }
 
+// SetExcludesRepo sets the ecludes for the repo <name> to [pkgs...]
+func (b *Builder) SetExcludesRepo(name, pkgs string) error {
```

Comment:
Maybe rename `name` to `repoName` to be more explicit about what it is naming. I had to look at the caller to figure out what that was.